ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -2243,6 +2243,22 @@ keys:
                     transit:
                       type: bool
                       description: Enable EVPN multicast transit mode.
+            evpn_multicast_gateway_dr_election_algorithm:
```

Comment:
Since command will accept only one algo hence we can update the model as per below:
```
          evpn_multicast_gateway_dr_election_algorithm:
              type: dict
              keys:
                algorithm:
                  type: str
                  required: true
                  description: Algorithm used to elect a gateway designated-router.
                  valid_values:
                    - hrw
                    - modulus
                    - preference
                preference_value:
                  type: int
                  description: Preference value for preference algorithm.
                  min: 0
                  max: 65535
                  convert_types:
                    - str
```

Reply:
fixed.